api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -153,25 +169,39 @@ message Attestation {
   bytes quorum_signed_percentages = 6;
 }
 
+// Global constant parameters defined by the payment vault.
 message PaymentGlobalParams {
+  // Global ratelimit for on-demand dispersals
   uint64 global_symbols_per_second = 1;
+  // Minimum number of symbols accounted for all dispersals
   uint32 min_num_symbols = 2;
+  // Price charged per symbol for on-demand dispersals
   uint32 price_per_symbol = 3;
+  // Reservation window for all reservations
   uint32 reservation_window = 4;
+  // quorums allowed to make on-demand dispersals
   repeated uint32 on_demand_quorum_numbers = 5;
 }
 
+// Reservation parameters of an account, used to determine the rate limit for the account.
 message Reservation {
+  // rate limit for the account
   uint64 symbols_per_second = 1;
+  // start timestamp of the reservation
   uint32 start_timestamp = 2;
+  // end timestamp of the reservation
   uint32 end_timestamp = 3;
+  // quorums allowed to make reserved dispersals
   repeated uint32 quorum_numbers = 4;
+  // quorum splits between allowed quorums
   repeated uint32 quorum_splits = 5;
 }
 
-// BinRecord is the usage record of an account in a bin. The API should return the active bin 
+// BinRecord is the usage record of an account in a bin. The API should return the active bin
```

Comment:
What is "bin"?

Reply:
Hope has since renamed this to `PeriodRecord`.